presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -92,6 +92,7 @@
     public static final String FILTER_AND_PROJECT_MIN_OUTPUT_PAGE_ROW_COUNT = "filter_and_project_min_output_page_row_count";
     public static final String USE_MARK_DISTINCT = "use_mark_distinct";
     public static final String PREFER_PARTITIAL_AGGREGATION = "prefer_partial_aggregation";
+    public static final String MAX_GROUPING_ELEMENTS = "max_grouping_elements";
```

Comment:
It's actually a limit on grouping sets, not on grouping elements. A grouping element is a syntactic construct such as a single "CUBE", "ROLLUP", etc. entry.